test/90_1_prm/data/prototype_list.json

```diff
@@ -28,6 +28,15 @@
         ["SmallOffice", "90.1-2013", "ASHRAE 169-2013-2A", "no_user_data",[]],
         ["Warehouse", "90.1-2013", "ASHRAE 169-2013-2A", "no_user_data",[]]
     ],
+    "vav_fan_curve": [
```

Comment:
We don't need this many tests. I would use MediumOffice in CZ 4A and also 2A, and SecondarySchool in 4A only. That gets PVAV, VAV, and also includes a case with parallel fan powered boxes.